ceph.spec.in

```diff
@@ -87,7 +87,13 @@
 
 %{!?_udevrulesdir: %global _udevrulesdir /lib/udev/rules.d}
 %{!?tmpfiles_create: %global tmpfiles_create systemd-tmpfiles --create}
+%if 0%{?rhel} == 7
+%{!?python3_pkgversion: %global python3_pkgversion 34}
+%{!?python3_version: %global python3_version 3.4}
```

Comment:
the "main" python3 in EPEL7 is now 3.6. see https://lists.fedoraproject.org/archives/list/epel-announce@lists.fedoraproject.org/message/EGUMKAIMPK2UD5VSHXM53BH2MBDGDWMO/ .

Reply:
I think building against 3.6 would be a step in the right direction given that rhel 7.7 might introduce python 3.6. Also, it should not matter what rhel minor version you use when you get the package from epel.

Reply:
@b-ranto do you mean we **always** use python3.6 as the default python3 for CentOS/RHEL7 build? or, probably we should add an option of `with_python3_other` to respect `python3_other_version`?

Reply:
@b-ranto #27417